deepchem/models/tests/test_layers.py

```diff
@@ -1283,24 +1283,25 @@ def test_FerminetElectronFeature():
     "Test for FerminetElectronFeature layer."
     electron_layer = dc.models.torch_models.layers.FerminetElectronFeature(
         [32, 32, 32], [16, 16, 16], 4, 8, 10, [5, 5])
-    one_electron_test = torch.randn(8, 10, 4 * 4)
-    two_electron_test = torch.randn(8, 10, 10, 4)
+    one_electron_test = torch.randn(8, 10, 4 * 4).double()
```

Comment:
Why are we doubling these? Can you add comments that explain the purpose?

Reply:
Removed double datatypes as per our previous conversation. (Thought double datatypes will improve precision, but slowed down the model)